projects/js-packages/connection/hooks/use-product-checkout-workflow/index.jsx

```diff
@@ -41,7 +41,7 @@ export default function useProductCheckoutWorkflow( {
 	} );
 
 	const initializeAnalytics = () => {
-		const tracksUser = syncSelect( STORE_ID ).getWpcomUser();
+		const tracksUser = isUserConnected ? syncSelect( STORE_ID ).getWpcomUser() : null;
 		const blogId = syncSelect( STORE_ID ).getBlogId();
```

Comment:
why do you continue running the following instructions? You should exit from here asap.

Reply:
Just force-pushed again with a different approach